packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -75,10 +109,10 @@ module.exports = function rollupLwcCompiler(pluginOptions = {}) {
                 // Could be an import like `import component from 'x/component'`
                 try {
                     return resolveModule(importee, importer, {
-                        modules: customResolvedModules,
-                        rootDir: customRootDir,
+                        modules,
+                        rootDir,
                     }).entry;
-                } catch (err) {
+                } catch (err: any) {
                     if (err.code !== 'NO_LWC_MODULE_FOUND') {
```

Comment:
```suggestion
                } catch (err: unkown) {
                    if (err && err.code !== 'NO_LWC_MODULE_FOUND') {
```

Reply:
The suggested change would have a TS error `Object is of type 'unknown'.` I'm not sure how to fix that. Maybe we can import `NoLwcModuleFound` from module-resolver and check using `!err instanceof NoLwcModuleFound` then throw. But that means exposing that error type so I'm not sure if it is worth it. Any other suggestions?